sklearn/linear_model/_omp.py

```diff
@@ -986,6 +1000,21 @@ class OrthogonalMatchingPursuitCV(RegressorMixin, LinearModel):
     array([-78.3854...])
     """
 
+    _parameter_constraints = {
+        "copy": ["boolean"],
+        "fit_intercept": ["boolean"],
+        "normalize": ["boolean", Hidden(StrOptions({"deprecated"}))],
+        "max_iter": [Interval(Integral, 0, None, closed="left"), None],
+        "cv": [
+            Interval(Integral, 2, None, closed="left"),
+            BaseCrossValidator,
```

Comment:
Technically the [documentation](https://scikit-learn.org/stable/glossary.html#term-CV-splitter) only specifies that it requires the `split` and `get_n_splits` methods, not that it inherits from `BaseCrossValidator`, so we can be a bit more general here.
```suggestion
            HasMethods(["split", "get_n_splits"]),
```